pyproject.toml

```diff
@@ -65,6 +75,9 @@ memory-profiler = "^0.61.0"
 
 [tool.poetry.extras]
 xarray = ["xarray", "numpy"]
+test = ["pytest", "pytest-benchmark", "pytest-randomly", "pytest-cov"]
+dev = ["xarray", "numpy", "pytest", "pytest-benchmark", "pytest-randomly",
+       "pytest-cov", "ruff", "pyyaml", "sphinx", "myst-parser", "sphinx-autoapi", "sphinx-rtd-theme"]
```

Comment:
Unfortunately poetry doesn't let you self-reference. Other backends let you do `dev = ["space_packet_parser[test,xarray]"` to combine other extras.

Reply:
Yeah, I'm falling out of love with Poetry as time goes on... If you want to explore other build backends, I'm open to it. Meson felt a little unfinished to me, at least wrt Rust, but maybe that's a better option and we can just use pip for local development?